students/ZidanLuo/session02/series.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+
+def fibonacci(n):
+	#list the simple cases for n
+	if n == 0 or n == 1:
+		return n
+	#recursion function
+	else:
+		return fibonacci(n-1) + fibonacci(n-2)
+
+
+def lucas(n):
+	#just like the fibonacci function, list the simple cases and 
+	#uses recursion
+	if n == 0:
+		return 2
+	elif n == 1:
+		return 1
+	else:
+		return lucas(n-1) + lucas(n-2)
+
+#n if the required parameter to determine which element in the series to print
+#y and z are optional parameters to determine the default values
+def sum_series(n, y = 0, z = 1):
```

Comment:
minor style issue, better to not use space around the `=` in parameters:

`def sum_series(n, y=0, z=1):`

See PEP8:

https://www.python.org/dev/peps/pep-0008/